contracts/LifCrowdsale.sol

```diff
@@ -126,10 +137,17 @@ contract LifCrowdsale is Ownable, Pausable {
     TokenPurchase(msg.sender, beneficiary, weiAmount, tokens);
   }
 
-  function addPresaleTokens(address beneficiary, uint256 tokens) onlyOwner {
+  function addPrivatePresaleTokens(address beneficiary, uint256 totalWei) onlyOwner {
     require(block.number < startBlock);
     require(beneficiary != address(0));
-    require(tokens > 0);
+    require(totalWei > 0);
```

Comment:
I think renaming `totalWei` to `weiSent` would be good, but for `totalPresaleWei` there is no need to rename it, that is the total wei raised by presale, and it cant be higher than `maxPresaleWei`